conf/cannon.config

```diff
@@ -0,0 +1,33 @@
+params{
+    config_profile_description = 'Harvard FAS Cannon Profile for running nextflow pipelines.'
+    config_profile_contact = 'Lei Ma (@microlei)'
+    config_profile_url = 'https://www.rc.fas.harvard.edu/'
+}
+
+singularity {
+    enabled = true
+    autoMounts = true
+}
+
+process {
+    executor = 'slurm'
```

Comment:
Maybe some [resourceLimits](https://www.nextflow.io/docs/latest/reference/process.html#process-resourcelimits)

Reply:
And also in the `params` you likely want to include the old `max_*` paramaeters to point to the same values so you can run older pipelines 

https://nf-co.re/docs/tutorials/use_nf-core_pipelines/writing_institutional_profiles#params-scope

(see in the [broken formatted] 'Note on older pipelines' box)